### ISSUE TITLE
It should note that bridge interface may need to be added to a firewall zone with firewalld

### ISSUE THREAD

**Issue by yryo617** [docker/docker.github.io#11691]
File: [network/network-tutorial-standalone.md](https://docs.docker.com/network/network-tutorial-standalone/)

This article should include warning that in some environments (e.g. clean firewalld on CentOS 8) user-defined bridge interface (which may be done via docker-compose) must be added to a zone using firewall-cmd.
Without a zone defined, firewalld will prohibit any communication between containers using that bridge.

**Comment by devZer0** [comment]
yes please!!! pulling my hair out for >1 hour now because of this !!!

**Comment by devZer0** [comment]
oh, btw - adding that interfaces manually via firewall-cmd is a workaround - not a solution.   the solution is, to integrate docker with firewalld on centos8.

i'm curious how centos8 is mentioned for usage when integration apparently is incomplete

https://docs.docker.com/engine/install/centos/#prerequisites

"To install Docker Engine, you need a maintained version of CentOS 7 or 8"


there should at least exist a hint with some warning

**Comment by rodrigogonegit** [comment]
This is happening to me after upgrading to docker `Docker version 27.1.2, build d01f264` and firewalld.

How to reproduce:
- Make sure iptables is disabled.
- Make sure firewalld is enabled
- Run these
```
docker network create --driver bridge mytestnet
docker run --net mytestnet --name terminal_a -it busybox sh
docker run --net mytestnet --name terminal_b -it busybox sh
```
- Get in the containers and try to ping. Observe it does not work.
- Disable firewalld. 
- Observe it works
- Aditionally, run `ip -br a` to save which interface points to the network you created above, in my case it is `br-281213661d1e`
- Run `sudo firewall-cmd --zone=docker --change-interface=br-281213661d1e` to add said interface to the docker zone
- Repeat ping. Observe it works.
